plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -934,6 +934,10 @@ func formRequest(reqData requestData, role *roleEntry, cl *endpoint.Connector, s
 		}
 	}
 
+	if role.ServerTimeout != 0 {
```

Comment:
According to a similar code in the VCert required change, this should be changed to `role.ServerTimeout > 0`

Reply:
solved by latest commit